scripts/fuzz_opt.py

```diff
@@ -448,6 +463,10 @@ def run_d8_wasm(wasm, liftoff=True):
     return run_d8_js(in_binaryen('scripts', 'fuzz_shell.js'), [wasm], liftoff=liftoff)
 
 
+def all_disallowed(features):
+    return not any([('--enable-' + x) in FEATURE_OPTS for x in features])
```

Comment:
```suggestion
    return not any(('--enable-' + x) in FEATURE_OPTS for x in features)
```

I think this does the same thing without materializing the list (and is slightly easier to read).